frontend_vue/src/views/HomeView.vue

```diff
@@ -1,9 +1,29 @@
 <template>
   <AppLayoutGrid>
     <div>Home</div>
+    <button @click="() => open()">Check modal</button>
   </AppLayoutGrid>
 </template>
 
 <script setup lang="ts">
 import AppLayoutGrid from '@/layout/AppLayoutGrid.vue';
+import { useModal } from 'vue-final-modal'
+import BaseModal from '@/components/base/_BaseModal.vue'
+
+  const { open, close } = useModal({
+    component: BaseModal,
+    attrs: {
+      title: 'My lovely modal',
+      onConfirm() {
+        close()
+      },
+    },
+    slots: {
+      default: 'Test modal content Test modal content Test modal content',
+      footer: `<button @click="onConfirm" class="btn btn-green">Footer content</button>`,
+    },
+  })
```

Comment:
this is kinda weird. is this just an example? Will modals that extend base have their own component files?

Reply:
I'm not sure why this is shown in the diff, it's marked as 'outdated' from github ( and it is in fact, I got rid of this code )
<img width="443" alt="Screenshot 2024-03-14 at 09 59 43" src="https://github.com/thinkst/canarytokens/assets/126554007/ec205083-fde9-49f9-b7d6-8a06062abb01">